docs/integrations/golang.md

```diff
@@ -81,7 +81,7 @@ Predefined build tags:
 #### `strategy: <string>`
 
 Manually specify the golang package manager being used. This should be untouched unless fossa is incorrectly attempting to analyze the wrong strategy. If this option is set, it is recommended [lockfile](#LockfilePath:-<string>) and [manifest](#ManifestPath:-<string>) be set as well. A list of supported strategies is as follows:
```

Comment:
Not that this should be changed now, but putting "This should be untouched unless fossa is incorrectly attempting to analyze the wrong strategy" in positive terms makes it much clearer: "You should change this only if fossa is attempting to analyze using the wrong strategy."